### DISCUSSION TITLE
List of posible shortForecast descriptions

### DISCUSSION THREAD

**Discussion by StjepanDzojic** [weather-gov/api#745] [Q&A] [answered]
Hi, I'm using the `/gridpoints/{wfo}/{x},{y}/forecast/hourly` endpoint and wanted to ask if there are a list of the shortForecast property. I need to display my custom icons, and it would be great if a have the list of every possible description. Thanks!

**Comment by dwhitemv25** [comment, marked as answer]
That question's been asked here before, so search the board for other answers, but I wanted to offer this option. In the Saratoga weather template scripts is code that converts keywords from the zone forecast to icons. Here is the link to the lookup table:

https://github.com/ktrue/NWS-forecast/blob/ccdfc4b0acf2598a1d9c5d500267be6362b6e0d5/advforecast2.php#L1747

You can use this as a starting point for your own parser.

**Reply by StjepanDzojic** [reply]
This is what i need. Thank you!
